### DISCUSSION TITLE
The app doesn't work

### DISCUSSION THREAD

**Discussion by dcn0907** [reflex-dev/reflex#3813] [General]
The api_url and the cors configuration is not been takken by the app, I'm deploying the frontend in vercel and the backend in render, when I try to run the app changing the url it doesn't break. It's steel working even in the local host.
import reflex as rx

config = rx.Config(
    app_name="link_bio",
    cors_allowed_origins=[
       "https://grone-web-bice.vercel.app/"
    ]
)

**Comment by picklelo** [comment]
I think this is an issue with how we're passing the CORS params. I'll convert this to an issue to track it: https://github.com/reflex-dev/reflex/issues/3819
